docs/content/docs/dev/datastream/sources.md

```diff
@@ -325,16 +325,13 @@ public class FixedFetcherSizeSourceReader<E, T, SplitT extends SourceSplit, Spli
         extends SourceReaderBase<E, T, SplitT, SplitStateT> {
 
     public FixedFetcherSizeSourceReader(
-            FutureCompletingBlockingQueue<RecordsWithSplitIds<E>> elementsQueue,
             Supplier<SplitReader<E, SplitT>> splitFetcherSupplier,
             RecordEmitter<E, T, SplitStateT> recordEmitter,
             Configuration config,
             SourceReaderContext context) {
         super(
-                elementsQueue,
                 new FixedSizeSplitFetcherManager<>(
-                        config.getInteger(SourceConfig.NUM_FETCHERS),
-                        elementsQueue,
+                        config.get(SourceConfig.NUM_FETCHERS),
                         splitFetcherSupplier),
```

Comment:
Configuration parameter was not passed to FixedSizeSplitFetcherManager.

Reply:
Maybe, I am missing something. It was actually passed :
```java
                new FixedSizeSplitFetcherManager<>(
                        config.getInteger(SourceConfig.NUM_FETCHERS),
```
what I did is changed the removed _getInteger_ to _get_. This code compiles now.

FixedSizeSplitFetcherManager expects an int as the first parameter in its constructor:

```java
    public FixedSizeSplitFetcherManager(
            int numFetchers,
```

Reply:
The construction method of FixedSizeSplitFetcherManager is
```
  public FixedSizeSplitFetcherManager(
            int numFetchers,
            Supplier<SplitReader<E, SplitT>> splitReaderSupplier,
            Configuration config)
```
but  here you created it with 
```
new FixedSizeSplitFetcherManager<>(
                        config.get(SourceConfig.NUM_FETCHERS),
                        splitFetcherSupplier)
```

Reply:
thank you for pointing it out. Fixed